frontend/src/app/ui/components/export-modal/export-modal.component.html

```
@@ -1,20 +1,20 @@
       <div class="form-group">
-        <label for="beatName">Beat</label>
+        <label for="beatName">{{ 'export.beat' | translate }}</label>
         <input id="beatName" type="text" [value]="beatName()" readonly class="beat-name-input" />
       </div>
 
       <div class="form-group">
-        <label for="loopCount">Duration (loops)</label>
+        <label for="loopCount">{{ 'export.duration' | translate }}</label>
         <select id="loopCount" [(ngModel)]="options.loopCount" class="form-select">
-          <option *ngFor="let count of loopCounts" [value]="count">{{ count }} loop{{ count > 1 ? 's' : '' }}</option>
+          <option *ngFor="let count of loopCounts" [value]="count">{{ count }} {{ count > 1 ? ('export.loops' | translate) : ('export.loop' | translate) }}</option>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)